### PR TITLE
Adding my repo to documentation index for disto

### PR DESCRIPTION
**Pull request by kejriwalnishant1990**

### CODE REVIEW

**Comment by kejriwalnishant1990** [comment]
I have provided the code on the url. Please rebuild it.

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -774,6 +774,17 @@ repositories:
...
+  bowpmap_ros:
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/kejriwalnishant1990/bowpmap_ros.git
+      version: 1.0
+    source:
+      type: git
+      url: https://github.com/kejriwalnishant1990/bowpmap_ros.git
+      version: indigo
```
Comment (tfoote) [inline indigo/distribution.yaml]: There is no `indigo` branch on this repository. The only current branch is `master`

File: indigo/distribution.yaml [hunk 2]
```diff
@@ -774,6 +774,17 @@ repositories:
       url: https://github.com/ros/bond_core.git
       version: master
     status: maintained
+  bowpmap_ros:
+    release:
```
Comment (tfoote) [inline indigo/distribution.yaml]: a release section requires you to have run a release using bloom. There are tutorials ehre: http://wiki.ros.org/bloom through that process bloom will open the pull request for you.

And please note that you should have a separate release repository from our default source repository.

**Review by tfoote — CHANGES_REQUESTED** [review]
At a minimum you need to fix the branch and remove the release tag. You can also add a doc entry for your repository as well.

**Pull request merged**